src/Freya.Core/Integration.fs

```diff
@@ -43,29 +39,26 @@ type OwinAppFunc =
 [<RequireQualifiedAccess>]
 [<CompilationRepresentation (CompilationRepresentationFlags.ModuleSuffix)>]
 module OwinAppFunc =
-    open Freya.Core.Functions
 
-    /// Converts a <see cref="Freya{T}" /> computation to an <see cref="OwinAppFunc" />.
-    [<CompiledName ("FromFreya")>]
-    let fromFreya (freya: Freya<_>) =
+    /// Converts a <see cref="Core{T}" /> computation to an <see cref="OwinAppFunc" />.
+    [<CompiledName ("FromCore")>]
+    let fromCore (core: Core<_>) =
```

Comment:
More than anything else, I think these changes highlight the problem with using `Core` as a type name. I prefer to leave these alone, using Freya in the name rather than Core or OWIN.